### DISCUSSION TITLE
Remove Survey Feedback

### DISCUSSION THREAD

**Discussion by joaosilva1002** [LiveHelperChat/livehelperchat#1820] [General]
Hi!

I need to make sure that this feedback message is not sent after the user completes the survey.

How can I make the chat close automatically after sending the survey?

![survey](https://user-images.githubusercontent.com/93102624/203628611-3b95ab37-5c15-47a7-8a34-0afd9e715a46.png)

**Comment by remdex** [comment]
Well, at the moment this scenario is not supported just. Unless you develop it or hire someone to do so :)
